services/oada_upload/dev-docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.1'
```

Comment:
What is the point of this file?

Reply:
I wish the author of the service had made this in the first place.  This file is what you'd copy to "docker-compose.yml" to dev on this service locally.  docker-compose.yml is .gitignored so that you can't accidently commit your local dev copy of it to the repo, but this is basically the "example" docker-compose.yml for a developer.

Reply:
> I wish the author of the service had made this in the first place.

Right here: https://github.com/oats-center/isoblue-avena/blob/master/services/oada_upload/docker-compose-example.yml

Reply:
@aaron97neu this dev-docker-composel.yml is used when developing locally on your machine rather than on an isoblue.  It maps the local filesystem in to /code so you can just edit and restart to see the container run, and doesn't depend on isoblue-specific other services like postgres.  i.e. I can just run a container on my machine and it will connect to a postgress on a running isoblue over wiregaurd before deploying to the isoblue.  The original example docker-compose is what you'd use when adding it to a docker-compose deployment.

What I meant by "I wish the author had made this in the first place" wasn't "man, @aaron97neu really dropped the ball :)" , I meant that we should make it standard procedure for isoblue services to have a dev docker-compose.yml file so you can easily just grab a service and start it on your machine straight away to dev on it when debugging.  You did a good job.